content/guides/ecommerce-connection/step0-who-is-your-connector-for.md

```diff
@@ -0,0 +1,53 @@
+# Who is your connector for?
+
+## Personnas
+
+You will find below user archetypes we use to help you guide your decisions about product features, interactions, and even visual design of your connector. By designing for the archetype — whose goals and behavior patterns are well understood — you can satisfy the broader group of people represented by that archetype.
+
+|![Julia](../../img/personas/Julia.png)|![Peter](../../img/personas/Peter.png)|![Filips](../../img/personas/Filips.png)|
+|:------------------------------------:|:------------------------------------:|:----------------------------------------------:|
+|     **Julia:** Product Manager       |        **Peter:** IT Manager         |    **Filips:** Julia/Peter's Integrator |
+
+
+## Personnas' needs
+
+You will find below the main needs, associated to each user profile, in terms of process, documentation, actions related to your future eCommerce connector.
+
+### Julia's needs
+
+::: info
```

Comment:
```suggestion
```
I'd remove the info block as they are not informative but totally part of the text.